tools/syz-reprolist/reprolist.go

```diff
@@ -111,3 +119,15 @@ func reproIDFromURL(url string) string {
 	}
 	return parts[1]
 }
+
+func saveBug(bug *api.Bug) error {
+	bugPath := path.Join(*flagOutputDir, bug.ID+".json")
```

Comment:
Do these names correspond to the C reproducer file names? If not, maybe it's better to introduce per-bug folders?

Reply:
No, they don't. We can have multiple reproducers for every bug.
I don't mind to folder them if you want.

Reply:
It's not so much that I want, but that storing these jsons alongside reproducers is barely any useful if we cannot match them to the reproducers. Folders sound like the simplest option to group them.

Reply:
It depends....

To find the reproducer 1469c9a8e80000.c bug:
```bash
$ grep -rl 1469c9a8e80000 .
./703fe1bb0d9043c4001d70abcbf129afe286249f.json
```

To count repros:
```bash
$ ll *.c | wc -l
4515
```

To count bugs:
```bash
$ ll *.json | wc -l
7732
```

Reply:
I plan this archive to be used by robots, not humans. The folder will not make a big diff.

Reply:
I was thinking about putting all bugs to "bugs" folder and all repros to "repros" folder to be able to add some README and some scripts.

Reply:
And there is an idea to add some metadata to every reproducer.
It'll be something like the information about origin+sanitizers.

Reply:
> `grep -rl 1469c9a8e80000 .`

But that is not convenient, isn't it?
If we aim for this to be used by robots which can handle many formats, why not pick the layout that is easier for humans (and that makes the implementation of those bots simpler)?

> I was thinking about putting all bugs to "bugs" folder and all repros to "repros" folder to be able to add some README and some scripts.

I meant the folder structure of the `/bug_id/description.json`, `/bug_id/repro_1.c` format or something similar to that.

Reply:
```bash
/bug_id1/description.json
/bug_id1/repro_1.c
/bug_id2/description.json
/bug_id2/repro_1.c
/bug_id3/description.json
```
VS
```bash
/bugs/bug_id1.json
/bugs/bug_id2.json
/bugs/bug_id3.json
/repros/repro_1.c
/repros/repro_2.c
```

What is better? :)
I don't think folder structure is a good place for metadata like bug_id...

Reply:
Let's ask for more opinions.
@dvyukov @ramosian-glider

Reply:
I don't understand what will use this and how, so can't suggest any design decisions. Maybe there are completely different options achieving the same end goal that don't involve this export at all. Hard to say.

Reply:
I'll make per-bug folders.